solr_sync.go

```diff
@@ -67,7 +68,7 @@ func (s *solrZkInstance) Listen() error {
 					shouldReconnect = isConnectionClosed(err)
 					log.Error(fmt.Errorf("[go-solr] error on nevent %v", nEvent))
 					sleepTime = backoff(sleepTime)
-					continue
+					break
```

Comment:
this will stop us from listening for zk updates.  the original intention was to not process events in the loop not to break out of it